src/client.ts

```diff
@@ -113,6 +113,20 @@ export class Client {
     });
   }
 
+  invoices(
+    params: {},
```

Comment:
as params we can pass in filters. do those need some type defenition?

Reply:
what kind of filters we support? passing limit of invoices that can be retrieved as params ?

Reply:
This is the list of supported params: 

https://guides.getalby.com/alby-wallet-api/reference/api-reference/invoices

![image](https://github.com/getAlby/js-sdk/assets/100827540/fdd22dab-eb36-4b65-b235-1e81170db41b)

Reply:
![image](https://github.com/getAlby/js-sdk/assets/55848322/3625dc1b-9f60-4ef4-b585-297ef7339654)

cool! but we didn't added support for params in api docs. we shall update that as well if this endpoint supports params

Reply:
we should update those, yes. can you do it? copy it from the other endpoint.

Reply:
I've updated gitbook: 

https://guides.getalby.com/alby-wallet-api/reference/api-reference/invoices#get-all-invoice-history

Reply:
I've also added some typed params for the invoices endpoint now.